pandas/tests/io/test_sql.py

```diff
@@ -1319,14 +1323,15 @@ def check(col):
     def test_date_parsing(self):
         # No Parsing
         df = sql.read_sql_table("types_test_data", self.conn)
-
+        # Now that GH 6415 is fixed, dates are automatically parsed to UTC
```

Comment:
write these reversed, IOW

dates are automatically parsed to UTC (GH 6415). A future reader likely won't look at the reference unless the comment is not good enough., the comment needs to stand on its own.